src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -358,6 +359,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // switch from a solid color brush to an acrylic one.
         _changeBackgroundColor(bg);
 
+        // Update selection markers
+        Windows::UI::Xaml::Media::SolidColorBrush selectionBackgroundBrush{ til::color{ newAppearance.SelectionBackground() } };
+        SelectionStartMarker().Fill(selectionBackgroundBrush);
+        SelectionEndMarker().Fill(selectionBackgroundBrush);

```

Comment:
```suggestion
        Windows::UI::Xaml::Media::SolidColorBrush cursorColorBrush{ til::color{ newAppearance.CursorColor() } };
        SelectionStartMarker().Fill(cursorColorBrush);
        SelectionEndMarker().Fill(cursorColorBrush);
```
I'm conflicted on which one we want. I could go either way tbh.
1. (current) marker colors == selection background
2. (suggestion) marker colors == cursor color

Reply:
@cinnamon-msft said cursor color so changing it to that.